designcompose/src/main/java/com/android/designcompose/squoosh/SquooshVariantTransition.kt

```diff
@@ -212,6 +260,12 @@ internal class SquooshVariantTransition {
             // This will wipe out any existing transition; we probably want to make a smooth
             // function here, like we do for the interaction-based transitions.
             if (toId != fromId) {
+                val transition =
+                    customVariantTransition?.let { it(VariantTransitionContext(view, variantView)) }
+                        ?: SmartAnimateTransition(
```

Comment:
minor nit: do you want to pull this out as a const somewhere?

Reply:
Done